packages/cli/test/integration/shared/testDb.ts

```diff
@@ -105,10 +107,17 @@ export async function terminate(testDbName: string) {
 export async function truncate(entities: Array<keyof IDatabaseCollections>, testDbName: string) {
 	const dbType = config.get('database.type');
 
+	const { pass: isShared, fail: isNotShared } = categorize(
+		entities,
+		(str: keyof IDatabaseCollections) => str.toLowerCase().startsWith('shared'),
+	);
+
+	const sortedEntities = [...isShared, ...isNotShared]; // shared tables must be cleared first
```

Comment:
nitpick: concat is faster in this case since the spread iterates over each item

ref: https://jsben.ch/qkIUR

```suggestion
	const sortedEntities = isShared.concat(isNotShared); // shared tables must be cleared first
```